board-api/src/main/java/com/example/board/domain/comment/controller/CommentController.java

```diff
@@ -0,0 +1,61 @@
+package com.example.board.domain.comment.controller;
+
+import com.example.board.domain.comment.dto.CommentCreateRequest;
+import com.example.board.domain.comment.dto.CommentResponse;
+import com.example.board.domain.comment.dto.CommentUpdateRequest;
+import com.example.board.domain.comment.service.CommentService;
+import com.example.board.global.security.util.SecurityUtil;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/comments")
+public class CommentController {
+
+    private final CommentService commentService;
+
+    @PostMapping("/{postId}")
+    public ResponseEntity<CommentResponse> createComment(
+        @PathVariable("postId") Long postId,
+        @Valid @RequestBody CommentCreateRequest commentCreateRequest
+    ){
+        Long currentUserId = SecurityUtil.getCurrentUserIdCheck();
```

Comment:
역할이 getCurrentUserId 이것 뿐인 것 같습니다?
```suggestion
        Long currentUserId = SecurityUtil.getCurrentUserId();
```

Reply:
이 부분은 말씀해주신 의도를 잘 이해하지 못한 것 같습니다.

Reply:
SecurityUtil. getCurrentUserIdCheck()에서 Check하는 역할이 애매하다고 생각들어서 메서드 명이 바꾸는 게 좋지 않을까 싶었습니다!  `getCurrentUserId` 메서드 안에 `.orElseThrow(() -> new CustomException(ACCESS_DENIED));`로직이 함께 들어가고 Check라는 이름을 삭제해도 되지 않을까요?